RAKE/RAKE.py

```diff
@@ -4,8 +4,11 @@
 # Automatic keyword extraction from indi-vidual documents.
 # In M. W. Berry and J. Kogan (Eds.), Text Mining: Applications and Theory.unknown: John Wiley and Sons, Ltd.
 
+from __future__ import absolute_import
 import re
 import operator
+import six
+from six.moves import range
```

Comment:
This one's a little weird, the desired behavior is actually overriding the range builtin to provide a range function that has the same behavior in both python2 and python3. I think `six.moves.range` is similar to `xrange` in python2 and `range` in python 3 (there is no `xrange` in python3). It returns a generator rather than a list.